components/suggestion/suggestion_box.jsx

```diff
@@ -231,10 +202,10 @@ export default class SuggestionBox extends React.Component {
             // iOS doesn't support e.relatedTarget, so we need to use the old method of just delaying the
             // blur so that click handlers on the list items still register
             if (this.presentationType !== 'date' || this.props.value.length === 0) {
-                this.handleEmitClearSuggestions(this.suggestionId, MIN_TIME_BEFORE_CLICK);
+                this.handleEmitClearSuggestions(MIN_TIME_BEFORE_CLICK);
```

Comment:
Can you add a note to the ticket to have QA test tapping on and around the autocomplete on iOS to make sure that it properly selects items when tapping on them and properly dismisses the autocomplete when you tap outside? This behaviour is finicky, so we should make sure that it's tested

Reply:
@lindy65 Can you make sure this works?